sdk/core/azure-core-serializer-json-gson/src/main/java/com/azure/core/serializer/json/gson/models/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * Package containing models that represent JSON using the Java class system.
+ */
+package com.azure.core.serializer.json.gson.models;
```

Comment:
Don't we need to export this package in `module-info.java` as well?

Reply:
Good catch